### DISCUSSION TITLE
Invisible Item Frames +

### DISCUSSION THREAD

**Discussion by itsMarkiS** [PurpurMC/Purpur#578] [__Dismissed__] [answered]
It would be a cool feature if every invisible item frame/glow item frame with no items emmits a configurable particle, it would be cool for decorations, creating fake particles or even just to know where are the invisible item frames

Configuration Examples:
the particle itself
the amount/color of particle
if it is a forced particle (displays even if the player has particle offs)

like this datapack does
https://www.planetminecraft.com/data-pack/invisible-item-frame-4506693/

**Comment by encode42** [comment, marked as answer]
This would be much easier (and configurable) via a plugin. I'm not aware of any existing solutions -- maybe @YouHaveTrouble can figure something out for [PurpurExtras](https://github.com/YouHaveTrouble/PurpurExtras)?

**Reply by YouHaveTrouble** [reply]
Sign me out, configurable particles are a menace.
If someone wants to Pr it into PE, sure, but I ain't doing it myself
